Extensions/BBText/JsExtension.js

```diff
@@ -0,0 +1,580 @@
+              _('Get ') + property.description,
+              _('Get ') + property.description,
+              _(''),
+              '',
+              `${property.iconPath}.png`,
+              `${property.iconPath}.png`
+            )
+            .addParameter(
+              'object',
+              _(`${objectName} object`),
+              objectName,
+              false
+            )
+            .getCodeExtraInformation()
+            .setFunctionName(`get${property.functionName}`);
+        }
+
+        // Add all the generic SETTERS
+        if (parameterType === 'number' || parameterType === 'string') {
+          gdObject
+            .addAction(
+              `Set${property.functionName}`,
+              _('Set ') + property.description,
+              _('Set ') + property.description,
+              _('Set ') +
+                property.description +
+                _(' of _PARAM0_ to _PARAM1_ _PARAM2_'),
+              '',
+              `${property.iconPath}.png`,
+              `${property.iconPath}.png`
+            )
+            .addParameter(
+              'object',
+              _(`${objectName} object`),
+              objectName,
+              false
+            )
+            .addParameter('operator', _("Modification's sign"), '', false)
+            .addParameter(parameterType, _(property.paramLabel), '', false)
+            .getCodeExtraInformation()
+            .setFunctionName(`set${property.functionName}`)
+            .setManipulatedType(parameterType)
+            .setGetter(`get${property.functionName}`);
+        } else {
+          // Setter doesnt have a +- modifier (Color, alignment, etc)
+          gdObject
+            .addAction(
+              `Set${property.functionName}`,
+              _('Set ') + property.description,
+              _('Set ') + property.description,
+              _('Set ') + property.description + _(' of _PARAM0_ to _PARAM1_'),
+              '',
+              `${property.iconPath}.png`,
+              `${property.iconPath}.png`
+            )
+            .addParameter(
+              'object',
+              _(`${objectName} object`),
+              objectName,
+              false
+            )
+            .addParameter(
+              parameterType,
+              _(property.paramLabel),
+              property.options ? property.options : '',
+              false
+            )
+            .getCodeExtraInformation()
+            .setFunctionName(`set${property.functionName}`)
+            .setGetter(`get${property.functionName}`);
+        }
+
+        // Add compare Conditions via getters
+        if (parameterType === 'string' || parameterType === 'number') {
+          const propExpressionType =
+            parameterType === 'string' ? 'string' : 'expression';
+          gdObject
+            .addCondition(
+              `Is${property.functionName}`,
+              _('Is the ') + property.description + _(' equal to a value'),
```

Comment:
```suggestion
              property.description
```

Conditions should be named just like the property: you don't want in the list of conditions to have tons of "Is XXX equal to a value". Remember that with the new editor, you have already selected the object, so you know that you're going to do some kind of tests on the object, now you just select the property to compare :)